pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1162,7 +1163,11 @@ func nodesWherePreemptionMightHelp(nodes []*v1.Node, failedPredicatesMap FailedP
 // considered for preemption.
 // We look at the node that is nominated for this pod and as long as there are
 // terminating pods on the node, we don't consider this for preempting more pods.
-func podEligibleToPreemptOthers(pod *v1.Pod, nodeNameToInfo map[string]*schedulernodeinfo.NodeInfo) bool {
+func podEligibleToPreemptOthers(pod *v1.Pod, nodeNameToInfo map[string]*schedulernodeinfo.NodeInfo, enableNonPreempting bool) bool {
+	if enableNonPreempting && *pod.Spec.PreemptionPolicy == v1.PreemptNever {
+		klog.V(5).Infof("Pod %v/%v is no eligible for preemption, because it uses a non-preempting priority class.", pod.Namespace, pod.Name)
```

Comment:
suggest `"Pod %v/%v is not eligible for preemption because it has a preemptionPolicy of %v", v1.PreemptNever`